docker/script/convert_merizo_unidoc_files.py

```diff
@@ -0,0 +1,49 @@
+import pandas as pd
+import os
+
+def process_merizo(chainsaw_file, merizo_file, output_file):
```

Comment:
I'm not clear why the merizo/unidoc are merged with chainsaw results?

Reply:
Okay, thanks. I think it would be more readable if the input/output files were given by command line parameters rather than being hardcoded here. I'll make the update